Sources/HAP/Base/Generated.swift

```diff
@@ -44,6 +44,47 @@ public enum AccessoryType: String, Codable {
 	case audioReceiver = "34"
 	case televisionSetTopBox = "35"
 	case televisionStreamingStick = "36"
+
+    public var description: String {
+        let descriptions = [
+			"1": "Other",
+			"2": "Bridge",
+			"3": "Fan",
+			"4": "Garage Door Opener",
+			"5": "Lightbulb",
+			"6": "Door Lock",
+			"7": "Outlet",
+			"8": "Switch",
+			"9": "Thermostat",
+			"10": "Sensor",
+			"11": "Security System",
+			"12": "Door",
+			"13": "Window",
+			"14": "Window Covering",
+			"15": "Programmable Switch",
+			"16": "Range Extender",
+			"17": "IP Camera",
+			"18": "Video Doorbell",
+			"19": "Air Purifier",
+			"20": "Air Heater",
+			"21": "Air Conditioner",
+			"22": "Air Humidifier",
+			"23": "Air Dehumidifier",
+			"24": "Apple TV",
+			"26": "Speaker",
+			"27": "AirPort",
+			"28": "Sprinkler",
+			"29": "Faucet",
+			"30": "Shower Head",
+			"31": "Television",
+			"32": "Target Controller",
+			"33": "Wi-Fi Router",
+			"34": "Audio Receiver",
+			"35": "Television Set Top Box",
+			"36": "Television Streaming Stick",
+        ]
+        return descriptions[self.rawValue] ?? ""
```

Comment:
Since these are generated files, rather than silently failing would it be better to `fatalError`?

The reason I ask is this seems like it would signal that there was an unexpected type and likely needs to be regenerated.

Reply:
Most certainly not. This file defines currently known and non-blacklisted services and characteristics only. It is highly likely others will be used and/or defined in the future that this package doesn't know about, at least yet.

The swifty way would be to return an optional, but this is a public standard API so I chose to return an empty string for unknown properties, which can be equally well tested for by the caller.